src/index.html

```diff
@@ -5,8 +5,412 @@
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="container">
+        <div class="card container__card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="img apple"
+            class="card__image"
+          >
+
+          <h2 class="card__content">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+            href="#buy"
+            class="card__bottom-buy"
+            data-qa="card-hover"
+          >
+          buy
+          </a>
+        </div>
+
+        <div class="card container__card" data-qa="card-hover">
+          <img
+            src="./images/imac.jpeg"
+            alt="img apple"
+            class="card__image"
+          >
+
+          <h2 class="card__content">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+
+            <p class="card__price-cost">$2,199</p>
+          </div>
+
+          <a
+          href="#buy"
+          class="card__bottom-buy"
+          data-qa="card-hover"
+        >
+          buy
+          </a>
```

Comment:
Pay attention on indents. Fix all similar places
```suggestion
          <a
            href="#buy"
            class="card__bottom-buy"
            data-qa="card-hover"
          >
          buy
          </a>
```